spec/system/space_ghost_spec.rb

```diff
@@ -56,7 +58,13 @@
       select another_user_str + "\u00A0", from: "user_find"
       click_on "Add Users"
 
-      # next step is to click on "add user" then figure out how to load in the user for the druple
+      # This part of the code is going through the rest of the fields of the wizard form to review and submit the project request
+      click_on "Next"
+      expect(page).to have_content("Enter the storage and access needs for your project.")
+      click_on "Next"
+      expect(page).to have_content("Review")
```

Comment:
FYI I think review is on every page (it's in the side bar).  A better check for the review page would be to test for below
```suggestion
      expect(page).to have_content("Take a moment to review your details ")
```